Makefile

```diff
@@ -1,26 +1,13 @@
-# Copyright 2017 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-PLATFORM       ?= linux/amd64
 REGISTRY_NAME  ?= index.docker.io/linode
 IMAGE_NAME     ?= linode-blockstorage-csi-driver
-REV            := $(shell git describe --long --tags --dirty)
+REV            := $(shell git describe --long --tags --dirty 2> /dev/null || echo "dev")
 IMAGE_VERSION  ?= $(REV)
 IMAGE_TAG      ?= $(REGISTRY_NAME)/$(IMAGE_NAME):$(IMAGE_VERSION)
 GOLANGCI_LINT_IMG := golangci/golangci-lint:v1.52-alpine
+RELEASE_DIR    ?= release
 
-export GO111MODULE=on
+.PHONY: ci
+ci: fmt vet lint test build
 
 .PHONY: fmt
 fmt:
```

Comment:
Not sure if this is the right target for `ci`, since it doesn't exit with a non-zero exit code if not formatted.

```
linode-blockstorage-csi-driver $ go fmt ./...
app/linode/main.go
linode-blockstorage-csi-driver $ echo $?
0
```

If anything... it prevents `lint` from finding out about these formatting issues since it formats code before the `lint` target is invoked.

Reply:
i honestly hate managing the `fmt` through make and would prefer people use a file watcher with goimports or something.. i would prefer to drop fmt entirely and ramp up the linter if you think that is a preferred workflow

Reply:
We'll have to check the `fmt` workflow is the default in golangci-lint (and if not, we should explicitly enable it). But again, in a separate PR.